circle.yml

```diff
@@ -20,3 +20,7 @@ test:
     override:
         - cd $GS_WD && go test -v $(glide novendor)
         - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
+        - cd $GS_WD &&
+          cp bindata.go bindata-repo.go &&
+          make update-bindata &&
+          cmp bindata.go bindata-repo.go
```

Comment:
I'll try to explain.

First, we can't generate `bindata.go` on the CI because it needs to be vendored in other repos, and for that to work we need the data to be commited to the repo.

https://github.com/giantswarm/k8scloudconfig/pull/18/commits/d8ef102f91db459a62c09441fb2e0efa76b29ccb makes bindata generation deterministic, that is, if the same data is in `master.yaml` and `worker.yaml`, the generated `bindata.go` file will not change.

We use this property to make sure the `bindata.go` that's submitted in the PR matches what's in `master.yaml` and `worker.yaml`.